packages/react/src/components/Masthead/README.md

```diff
@@ -37,40 +37,42 @@ MASTHEAD_L1=true
 > [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/.env.example)
 > for more information
 
-## Navigation Types
+## navigation (String | Object | null)
```

Comment:
May be a bit silly, but shouldn't we use backticks like `navigation` and `platform`?

Reply:
I'm fine with that but as long as we stay consistent. I don't think we backtick other README props that are headlines. [For example](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/components/Layout)

We can probably do a huddle topic on README formatting, and see if there's a consistent approach we'd like to take.

Reply:
@emerson-pereira added as a huddle topic for next week.